src/pages/Dashboard/Leaderboard.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { useQuery, gql } from '@apollo/client';
+import { Box, Flex, VStack, Heading, Text, Table, Tr, Td, Th, Thead, Tbody } from '@chakra-ui/react';
+import { useSelector } from 'react-redux';
+
+const GET_GLOBAL_LEADERBOARD = gql`
+  query getGlobalLeaderboard {
+    getGlobalLeaderboard {
+      id
+      name
+      organization
+      itemsSold
+    }
+  }
+`;
+
+const composeTableHeader = (header, i) => {
+  return (
+    <Th bgColor="white" key={i}>
+      <Heading textTransform="uppercase" as="p" size="subtitle" color="black" opacity="0.5" mb="8px">
+        {header}
+      </Heading>
+    </Th>
+  );
+};
+
+const TableRow = props => {
+  const { bgColor, borderBottomWidth, rank, teamName, affiliation, itemsSold } = props;
+  return (
+    <Tr key={rank} borderBottomWidth={borderBottomWidth} borderColor="white" bg={bgColor} height="88px">
+      <Td>
+        {(rank === 1 && (
+          <Text fontSize="40px">
+            <span role="img" aria-label="gold medal">
+              🥇
+            </span>
+          </Text>
+        )) ||
+          (rank === 2 && (
+            <Text fontSize="40px">
+              <span role="img" aria-label="gold medal">
+                🥈
+              </span>
+            </Text>
+          )) ||
+          (rank === 3 && (
+            <Text fontSize="40px">
+              <span role="img" aria-label="gold medal">
+                🥉
+              </span>
+            </Text>
+          )) || (
+            <Text opacity="0.5" paddingLeft="6px">
+              {rank}
+            </Text>
+          )}
+      </Td>
+      <Td>
+        <Heading size="h4">{teamName}</Heading>
+      </Td>
+      <Td>
+        <Text size="body">{affiliation}</Text>
+      </Td>
+      <Td fontWeight="semibold">{itemsSold}</Td>
+    </Tr>
+  );
+};
+
+const composeTableBody = (row, i) => {
+  const { name, organization, itemsSold } = row;
+  const colors = ['#FFF9DA', '#FAF7F9', '#FFF2E7'];
+  return (
+    <TableRow
```

Comment:
react is throwing an warning, I believe key needs to be passed here (and not in `Tr`). Also, it's a good idea to use `id` as the key